src/native-stack/views/NativeStackView.tsx

```diff
@@ -42,18 +44,52 @@ export default function NativeStackView({
           contentStyle,
         } = options;
 
+        const viewStyles = [
+          styles.container,
+          stackPresentation !== 'transparentModal' && {
+            backgroundColor: colors.background,
+          },
+          contentStyle,
+        ];
+
         return (
           <Screen
             key={route.key}
             style={StyleSheet.absoluteFill}
-            gestureEnabled={Platform.OS === 'android' ? false : gestureEnabled}
+            gestureEnabled={isAndroid ? false : gestureEnabled}
             stackPresentation={stackPresentation}
             stackAnimation={stackAnimation}
+            onWillAppear={() => {
```

Comment:
@WoLewicki these extra methods were causing the merge conflict but i don't think they exist any longer, especially the emit type options. Only 'appear' and 'dismiss' are available. If i'm to remove then and only keep `onAppear` and `onDismissed` it then says it is a merge conflict.

Reply:
Navigation events where recently added and you can see them on `master` https://github.com/software-mansion/react-native-screens/blob/master/src/native-stack/views/NativeStackView.tsx. I don't know why they appear as changes added by you. Did you pull the master branch before rebasing?